x-pack/plugins/ml/public/application/jobs/jobs_list/components/jobs_stats_bar/jobs_stats_bar.js

```diff
@@ -94,17 +100,20 @@ function createJobStats(jobsSummaryList) {
     jobStats.failed.show = false;
   }
 
-  jobStats.activeNodes.value = Object.keys(mlNodes).length;
+  if (displayNodeInfo) {
+    jobStats.activeNodes.value = Object.keys(mlNodes).length;
+  }
 
   return jobStats;
 }
 
-export const JobStatsBar = ({ jobsSummaryList }) => {
-  const jobStats = createJobStats(jobsSummaryList);
+export const JobStatsBar = ({ jobsSummaryList, isServerless }) => {
+  const jobStats = createJobStats(jobsSummaryList, isServerless);
```

Comment:
I reckon job stats should be adjusted server-side as well

Reply:
This flag is for hiding the active ml nodes count section of the stats bar. The count will be 0 due to other changes which remove the node info from the jobs, but the label still needs to be hidden if running in serverless.